nav2_collision_monitor/src/polygon.cpp

```diff
@@ -35,7 +35,8 @@ Polygon::Polygon(
   const std::string & base_frame_id,
   const tf2::Duration & transform_tolerance)
 : node_(node), polygon_name_(polygon_name), action_type_(DO_NOTHING),
-  slowdown_ratio_(0.0), footprint_sub_(nullptr), tf_buffer_(tf_buffer),
+  slowdown_ratio_(0.0), limit_x_(0.0), limit_y_(0.0), limit_tw_(0.0),
```

Comment:
The same: I do not think, we need to have separate `limit_x` and `limit_y` values. It is more reasonable to have `limit_linear` and `limit_tw` values.